src/otaclient/ota_core.py

```diff
@@ -536,40 +545,45 @@ def _execute_update(self):
 
         # ------ pre-update ------ #
         logger.info("enter local OTA update...")
+
+        with TemporaryDirectory() as _tmp_dir:
+            use_inplace_mode = can_use_in_place_mode(
+                dev=self._boot_controller.standby_slot_dev,
+                mnt_point=_tmp_dir,
+                threshold_in_bytes=int(
+                    self._ota_metadata.total_regulars_size
+                    * STANDBY_SLOT_USED_SIZE_THRESHOLD
+                ),
+            )
```

Comment:
Prepare a temporary directory for `can_use_in_place_mode` to mount the standby slot for checking over it.